lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -97,11 +97,14 @@ function SlurpOutput($file, $cur_line, $out_type) {
 
 $err_cur_line = 0
 $out_cur_line = 0
+$timeout = <%= execution_timeout * 10 %>
+$sec = 0
 do {
   Start-Sleep -m 100
   $out_cur_line = SlurpOutput $out_file $out_cur_line 'out'
   $err_cur_line = SlurpOutput $err_file $err_cur_line 'err'
-} while (!($registered_task.state -eq 3))
+  $sec++
+} while( (!($registered_task.state -eq 3)) -and ($sec -lt $timeout) )
```

Comment:
While it should be close, this seconds based timeout is not reliable. Rather than counting iterations we should diff the current time and the time when the execution started.

Reply:
I will do that.

Reply:
Done